init.sql

```diff
@@ -159,3 +159,128 @@ create table asset_creators
+    candy_machine_id           bytea references candy_machine (id),
+    mode                       whitelist_mint_mode,                                      
+    whitelist_mint             bytea,                                                    
+    presale                    bool,                                                    
+    discount_price             int,
+    prefix_name                varchar(10),                             
+    name_length                int,                                     
+    prefix_uri                 varchar(10),                            
+    uri_length                 int,                                    
+    is_sequential              bool,
+    gatekeeper_network         bytea,                                    
+    expire_on_use              bool,  
+    number                     int,                                     
+    end_setting_type           end_setting_type,    
+    name                       varchar(50),                              
+    uri                        varchar(200),                             
+    hash                       bytea                                                                    
+);
+create unique index candy_machine_data_candy_machine_id on candy_machine_data (candy_machine_id);
+
+create table candy_machine_creators
+(
+    id                    bigserial                                PRIMARY KEY,
+    candy_machine_id      bytea references candy_machine (id)      not null,
+    creator               bytea                                    not null,
+    share                 int                                      not null default 0,
+    verified              bool                                     not null default false
+);
+create unique index candy_machine_creators_candy_machine_id on candy_machine_creators (candy_machine_id);
+create index candy_machine_creator on candy_machine_creators (candy_machine_id, creator);
+create index candy_machine_verified_creator on candy_machine_creators (candy_machine_id, verified);
+
+create table candy_guard
+(   
+    id                   bytea                                   PRIMARY KEY,
+    bump                 int                                     not null,
+    authority            bytea                                   not null,
+)
+
+create table candy_guard_group
+(
+    id                   bigserial                              PRIMARY KEY,
+    label                varchar(50),
+    candy_guard_id       bytea references candy_guard (id)      not null,
+    mode                 whitelist_mint_mode,                                      
+    whitelist_mint       bytea,                                                    
+    presale              bool,                                                    
+    discount_price       int,
+    lamports             int,
+    last_instruction     bool,
+    amount               int,
+    destination          bytea,
```

Comment:
This is a bit confusing now.  I happen to know that `lamports` and `last_instruction` came from the `BotTax` guard, and `amount` and `destination` came from the Lamports guard.  But you can't tell from this table any more.  Maybe all these guard fields should be prefixed with the guard they came from i.e. `bot_tax_lamports`, or `live_date` instead of just `date`, `allow_list_merkle_root`, etc.?

Reply:
im going to update them to be more descriptive of their settings/guard types today